src/style.css

```
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
```

You should use one font-family: Avenir and set different weight for them, so later you can have a default font-family for whole document and just set font-weight on the element (as you are doing)
```suggestion
@font-face {
  font-family: "Avenir";
  src: url(fonts/Avenir-Book.ttf);
  font-weight: 300;
}

@font-face {
  font-family: "Avenir";
  src: url(fonts/Avenir-Heavy.ttf);
  font-weight: 900;
}
```